### ISSUE TITLE
Allow mods to see all reports in a game they are in or are spectating

### ISSUE THREAD

**Issue by nrabins** [tagpro-game/tagpro-issues#146]
This would allow mods to issue warnings / see false reporting in the flow of the game, instead of having to open up mod tools, find the recent reports, see which players did what, and then return to the game to issue warnings.

**Comment by DuudeTP** [comment]
This already happens for games mods are spectating, at least.

**Comment by nrabins** [comment]
Only the player you're spectating, and only in spectator mode. I'd like this functionality all of the time, as I don't often spectate players unless I'm already very suspicious of them.

**Comment by DuudeTP** [comment]
No, you can now see all the reports in a game you're spectating, even if you aren't spectating the actual player reporting. This was added when 3.0 came around. I agree with your other point though, just making it clear that part of what you said has been implemented already.

**Comment by nrabins** [comment]
Oh, I suppose that's confirmation bias at work for me then, since I'm
usually spectating the person that would be issuing more reports.
